### ISSUE TITLE
Dependency collector parsing deprecation

### ISSUE THREAD

**Issue by theavey** [rdkit/homebrew-rdkit#53]
Updating homebrew today with `brew update` gives me a few warnings about deprecated dependency parsing:

```
Warning: Calling 'depends_on :python3' is deprecated!
Use 'depends_on "python3"' instead.
/usr/local/Homebrew/Library/Homebrew/dependency_collector.rb:88:in `parse_spec'

Warning: Calling 'depends_on :postgresql' is deprecated!
Use 'depends_on "postgresql"' instead.
/usr/local/Homebrew/Library/Taps/rdkit/homebrew-rdkit/Formula/rdkit.rb:24:in `<class:Rdkit>'
Please report this to the rdkit/rdkit tap!

Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/rdkit/homebrew-rdkit/Formula/rdkit.rb:32:in `<class:Rdkit>'
Please report this to the rdkit/rdkit tap!```

**Comment by mcs07** [comment]
Should be fixed by #52

**Comment by zmactep** [comment]
Yes, #52 fixes this. Please merge the PR.
